compiler/luci/import/include/luci/Import/GraphBuilderRegistry.h

```diff
@@ -32,6 +33,11 @@ struct GraphBuilderSource
    * @brief Returns registered GraphBuilder pointer for operator (nullptr if not present)
    */
   virtual const GraphBuilderBase *lookup(const circle::BuiltinOperator &op) const = 0;
+
+  /**
+   * @brief Returns registered SpecialNodeBuilder pointer for type (nullptr if not present)
+   */
+  virtual const SpecialNodeBuilder *lookup(SpecialNodeBuilderType type) const = 0;
```

Comment:
About word `Special`. I don't think using `Special` that is ambigious in purpose is good to use. Please use specific purpose. If this is to provide a tensor with data, please provide a word that relfects that purpose.

Reply:
I like the idea of introducing `~~~NodeBuilder`.

Reply:
To follow existing `GraphBuilderBase` name, I think `NodeBuilderBase` would be OK.